packages/browser/src/transports/xhr.ts

```diff
@@ -56,6 +56,7 @@ export class XHRTransport extends BaseTransport {
 
             request.open('POST', sentryRequest.url);
             for (const header in this.options.headers) {
+              // eslint-disable-next-line no-prototype-builtins
               if (this.options.headers.hasOwnProperty(header)) {
```

Comment:
Any reason not to just fix this rather than ignore it?

```suggestion
              if (Object.prototype.hasOwnProperty.call(this, header)) {
```

Reply:
Wanted to make the code change as minimal as possible. I'll come back and address these low hanging fruits in separate PRs.